tests-lists/openshift_network_stress.txt

```diff
@@ -0,0 +1,26 @@
+"[sig-network] DNS should support configurable pod DNS nameservers [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]"
```

Comment:
Please rename the file to `openshift-network-stress.txt`

Reply:
Done